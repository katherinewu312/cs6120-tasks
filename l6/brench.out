benchmark,run,result
quadratic,baseline,785
quadratic,crude-roundtrip,5309
montgomery,baseline,17
montgomery,crude-roundtrip,47
primes-between,baseline,574100
primes-between,crude-roundtrip,incorrect
mod_pow,baseline,243
mod_pow,crude-roundtrip,1408
orders,baseline,5352
orders,crude-roundtrip,incorrect
sum-check,baseline,5018
sum-check,crude-roundtrip,17042
palindrome,baseline,298
palindrome,crude-roundtrip,1820
rot13,baseline,8
rot13,crude-roundtrip,21
totient,baseline,253
totient,crude-roundtrip,1656
fib_recursive,baseline,2693
fib_recursive,crude-roundtrip,18218
relative-primes,baseline,1923
relative-primes,crude-roundtrip,18776
hanoi,baseline,99
hanoi,crude-roundtrip,310
is-decreasing,baseline,127
is-decreasing,crude-roundtrip,719
check-primes,baseline,8468
check-primes,crude-roundtrip,70463
sum-sq-diff,baseline,3038
sum-sq-diff,crude-roundtrip,13993
fitsinside,baseline,10
fitsinside,crude-roundtrip,18
hamming,baseline,117
hamming,crude-roundtrip,365
fact,baseline,229
fact,crude-roundtrip,861
loopfact,baseline,116
loopfact,crude-roundtrip,564
recfact,baseline,104
recfact,crude-roundtrip,481
geometric-sum,baseline,35
geometric-sum,crude-roundtrip,187
factors,baseline,72
factors,crude-roundtrip,423
perfect,baseline,232
perfect,crude-roundtrip,1856
binpow,baseline,105
binpow,crude-roundtrip,428
bitshift,baseline,167
bitshift,crude-roundtrip,933
digital-root,baseline,247
digital-root,crude-roundtrip,836
up-arrow,baseline,252
up-arrow,crude-roundtrip,1357
mccarthy91,baseline,1385
mccarthy91,crude-roundtrip,3981
sum-divisors,baseline,159
sum-divisors,crude-roundtrip,956
permutation,baseline,130
permutation,crude-roundtrip,603
combination,baseline,178
combination,crude-roundtrip,630
ackermann,baseline,1464231
ackermann,crude-roundtrip,7669129
pythagorean_triple,baseline,61518
pythagorean_triple,crude-roundtrip,339483
euclid,baseline,563
euclid,crude-roundtrip,2160
bbs,baseline,137
bbs,crude-roundtrip,367
binary-fmt,baseline,100
binary-fmt,crude-roundtrip,299
lcm,baseline,2326
lcm,crude-roundtrip,6628
gcd,baseline,46
gcd,crude-roundtrip,280
catalan,baseline,659378
catalan,crude-roundtrip,4231827
armstrong,baseline,133
armstrong,crude-roundtrip,556
pascals-row,baseline,146
pascals-row,crude-roundtrip,644
collatz,baseline,169
collatz,crude-roundtrip,977
sum-bits,baseline,73
sum-bits,crude-roundtrip,229
sqrt_bin_search,baseline,744
sqrt_bin_search,crude-roundtrip,5640
rectangles-area-difference,baseline,14
rectangles-area-difference,crude-roundtrip,36
mod_inv,baseline,558
mod_inv,crude-roundtrip,3961
karatsuba,baseline,1548
karatsuba,crude-roundtrip,7729
reverse,baseline,46
reverse,crude-roundtrip,274
fizz-buzz,baseline,3652
fizz-buzz,crude-roundtrip,43428
bitwise-ops,baseline,1690
bitwise-ops,crude-roundtrip,7948
sum-digits,baseline,219
sum-digits,crude-roundtrip,967
delannoy,baseline,5748752
delannoy,crude-roundtrip,timeout
