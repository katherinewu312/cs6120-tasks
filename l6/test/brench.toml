extract = 'total_dyn_inst: (\d+)'
benchmarks = 'benchmarks/*/*.bril'

[runs.baseline]
pipeline = [
    "bril2json",
    "brili -p {args}",
]

[runs.lvn]
pipeline = [
    "bril2json",
    "python3 ../to_ssa.py",
    "brili -p {args}",
]
