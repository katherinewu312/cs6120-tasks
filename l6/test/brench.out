benchmark,run,result
quadratic,baseline,785
quadratic,to_ssa,8267
montgomery,baseline,17
montgomery,to_ssa,55
primes-between,baseline,574100
primes-between,to_ssa,timeout
mod_pow,baseline,243
mod_pow,to_ssa,2092
orders,baseline,5352
orders,to_ssa,timeout
sum-check,baseline,5018
sum-check,to_ssa,25050
palindrome,baseline,298
palindrome,to_ssa,2690
rot13,baseline,8
rot13,to_ssa,25
totient,baseline,253
totient,to_ssa,2475
fib_recursive,baseline,2693
fib_recursive,to_ssa,24298
relative-primes,baseline,1923
relative-primes,to_ssa,28488
hanoi,baseline,99
hanoi,to_ssa,398
is-decreasing,baseline,127
is-decreasing,to_ssa,1083
check-primes,baseline,8468
check-primes,to_ssa,110683
sum-sq-diff,baseline,3038
sum-sq-diff,to_ssa,21265
fitsinside,baseline,10
fitsinside,to_ssa,18
hamming,baseline,117
hamming,to_ssa,479
fact,baseline,229
fact,to_ssa,1071
loopfact,baseline,116
loopfact,to_ssa,852
recfact,baseline,104
recfact,to_ssa,661
geometric-sum,baseline,35
geometric-sum,to_ssa,283
factors,baseline,72
factors,to_ssa,630
perfect,baseline,232
perfect,to_ssa,2850
binpow,baseline,105
binpow,to_ssa,545
bitshift,baseline,167
bitshift,to_ssa,1269
digital-root,baseline,247
digital-root,to_ssa,1124
up-arrow,baseline,252
up-arrow,to_ssa,2065
mccarthy91,baseline,1385
mccarthy91,to_ssa,4846
sum-divisors,baseline,159
sum-divisors,to_ssa,1359
permutation,baseline,130
permutation,to_ssa,884
combination,baseline,178
combination,to_ssa,789
ackermann,baseline,1464231
ackermann,to_ssa,9996529
pythagorean_triple,baseline,61518
pythagorean_triple,to_ssa,479037
euclid,baseline,563
euclid,to_ssa,3084
bbs,baseline,137
bbs,to_ssa,479
binary-fmt,baseline,100
binary-fmt,to_ssa,384
lcm,baseline,2326
lcm,to_ssa,8977
gcd,baseline,46
gcd,to_ssa,424
catalan,baseline,659378
catalan,to_ssa,5963920
armstrong,baseline,133
armstrong,to_ssa,793
pascals-row,baseline,146
pascals-row,to_ssa,956
collatz,baseline,169
collatz,to_ssa,1513
sum-bits,baseline,73
sum-bits,to_ssa,313
sqrt_bin_search,baseline,744
sqrt_bin_search,to_ssa,8072
rectangles-area-difference,baseline,14
rectangles-area-difference,to_ssa,46
mod_inv,baseline,558
mod_inv,to_ssa,6107
karatsuba,baseline,1548
karatsuba,to_ssa,11005
reverse,baseline,46
reverse,to_ssa,406
fizz-buzz,baseline,3652
fizz-buzz,to_ssa,69916
bitwise-ops,baseline,1690
bitwise-ops,to_ssa,11094
sum-digits,baseline,219
sum-digits,to_ssa,1451
delannoy,baseline,5748752
delannoy,to_ssa,timeout
cholesky,baseline,3761
cholesky,to_ssa,46685
mat-inv,baseline,1044
mat-inv,to_ssa,2943
gol,baseline,1425
gol,to_ssa,8533
dead-branch,baseline,1196
dead-branch,to_ssa,9974
function_call,baseline,timeout
function_call,to_ssa,timeout
ray-sphere-intersection,baseline,142
ray-sphere-intersection,to_ssa,276
birthday,baseline,484
birthday,to_ssa,3736
conjugate-gradient,baseline,1999
conjugate-gradient,to_ssa,14878
leibniz,baseline,12499997
leibniz,to_ssa,timeout
n_root,baseline,733
n_root,to_ssa,4088
newton,baseline,217
newton,to_ssa,855
euler,baseline,1908
euler,to_ssa,12424
riemann,baseline,298
riemann,to_ssa,1892
mandelbrot,baseline,2720947
mandelbrot,to_ssa,timeout
logistic,baseline,1110
logistic,to_ssa,4734
norm,baseline,505
norm,to_ssa,3038
cordic,baseline,517
cordic,to_ssa,40454
exponentiation-by-squaring,baseline,187
exponentiation-by-squaring,to_ssa,2497
pow,baseline,36
pow,to_ssa,230
ray-bbox-intersection,baseline,33
ray-bbox-intersection,to_ssa,861
sqrt,baseline,322
sqrt,to_ssa,5398
quickselect,baseline,279
quickselect,to_ssa,2132
1dconv,baseline,391
1dconv,to_ssa,4319
sieve,baseline,3482
sieve,to_ssa,23026
bubblesort,baseline,253
bubblesort,to_ssa,1804
random_walk,baseline,516754
random_walk,to_ssa,3645210
primitive-root,baseline,11029
primitive-root,to_ssa,67483
adler32,baseline,6851
adler32,to_ssa,32027
fnv1-hash,baseline,1768440
fnv1-hash,to_ssa,12357535
adj2csr,baseline,56629
adj2csr,to_ssa,411095
char-poly,baseline,383
char-poly,to_ssa,880
csrmv,baseline,121202
csrmv,to_ssa,1022535
dot-product,baseline,88
dot-product,to_ssa,267
shufflesort,baseline,36564
shufflesort,to_ssa,308734
major-elm,baseline,47
major-elm,to_ssa,451
cordic,baseline,1062
cordic,to_ssa,16732
max-subarray,baseline,193
max-subarray,to_ssa,837
mat-mul,baseline,1990407
mat-mul,to_ssa,16264719
fib,baseline,121
fib,to_ssa,765
vsmul,baseline,86036
vsmul,to_ssa,319606
quicksort-hoare,baseline,27333
quicksort-hoare,to_ssa,178599
quicksort,baseline,264
quicksort,to_ssa,1621
two-sum,baseline,98
two-sum,to_ssa,784
eight-queens,baseline,1006454
eight-queens,to_ssa,11494731
connected-components,baseline,1978
connected-components,to_ssa,26271
binary-search,baseline,78
binary-search,to_ssa,600
